### ISSUE TITLE
TypeError: plugin.apply is not a function

### ISSUE THREAD

**Issue by arve0** [valeriangalliat/markdown-it-anchor#12]
```sh
...\node_modules\markdown-it\lib\index.js:477
  plugin.apply(plugin, args);
         ^

TypeError: plugin.apply is not a function
```

Seems like the babel upgrade uses `exports.default`, which node does not load as a function:

```sh
$ npm install markdown-it-anchor
$ node
> require("markdown-it-anchor")
{ default:
   { [Function: anchor]
     defaults:
      { level: 1,
        slugify: [Function: slugify],
        permalink: false,
        renderPermalink: [Function: renderPermalink],
        permalinkClass: 'header-anchor',
        permalinkSymbol: '¶',
        permalinkBefore: false } } }
```

Which breaks `md.use(require('markdown-it-anchor'))`.

**Comment by valeriangalliat** [comment]
Fixed as of 2.3.2, thank you!

**Comment by fzdy1914** [comment]
It seems that this problem is back again.
